src/templates/app/trade/explanation.jsx

```diff
@@ -169,6 +194,36 @@ const Explanation = () => (
                 <p>{it.L('The <strong>end time</strong> is the selected number of minutes/hours after the <strong>start time</strong> (if less than one day in duration), or at the end of the trading day (if one day or more in duration).')}</p>
                 <p className='hint'><strong>{it.L('Note')}: </strong>{it.L('Touch/No Touch contracts will be refunded at the purchase price if there are less than 2 ticks between the start and end times.')}</p>
             </div>
+            <div id='explain_lookbacklow' className='invisible'>
+                <h3>{it.L('High, Low and Close')}</h3>
+                <p>{it.L('The [_1]high[_2] is the highest point ever reached by the market during the contract period.', '<strong>','</strong>')}</p>
+                <p>{it.L('The [_1]low[_2] is the lowest point ever reached by the market during the contract period.', '<strong>','</strong>')}</p>
```

Comment:
@4p00rv Using placeholder is not consistent with how we have handled `<strong>` so far. Translators got confused as well. is there any reason or advantage we use a placeholder here?